feat: complete Awareness section implementation

Rofieda-Maamar opened the pull request:
awareness section

Rofieda-Maamar committed:
feat: complete Awareness section implementation

belaid1meriem reviewed (changes requested):
**Major Standards Violation - Image Implementation:** The component uses native `<img>` tags instead of the Next.js `Image` component, missing optimization benefits and directly violating our coding standards. Replace all `<img>` tags with the Next.js `Image` component.

**Required changes:** Import `Image` from `next/image`, add proper width and height props, include the `sizes` prop for responsive optimization, and add `aria-hidden="true"` to decorative images.

**Security Violation - dangerouslySetInnerHTML:** Using `dangerouslySetInnerHTML` for line breaks poses security risks and violates best practices. Remove `dangerouslySetInnerHTML` and use the CSS `white-space` property instead. Update the description data to use `"\n"` for line breaks instead of `<br />` tags, then apply the `whitespace-pre-line` className to properly render line breaks.

**Color System Violation:** Uses `bg-[#F8598F]` instead of the standardized `bg-pink` from the design system defined in the theme. Replace all hardcoded color values with the theme variable.

**Missing Data File:** The reasons array is defined inline within the component file, which violates project structure standards. Create a separate `awareness-data.ts` file in the data folder following the pattern used in `objectives-data.ts`, `breast-cancer-data.tsx`, and other data files. Define proper TypeScript interfaces (`AwarenessReason` interface) with semantic property naming, and export the data array for import in the component.

**Cramped Layout on Medium Screens:** Keep a column layout on medium screens and make it a row on large screens and above.

**Image Positioning Classes:** The classNames and positioning logic are unnecessarily complex and difficult to maintain. Follow these tips for consistent image positioning:

- Give it a semantic name (not pic1, pic2, …).
- After exporting the image, make sure the bounds of the image match the actual content and that there is no empty space that could break the layout. If there is empty space, crop the image to the exact content size.
- Use the `fill` property in the `Image` (from `next/image`) and `object-contain` in `className` so the image fills the container instead of using arbitrary sizing.
- avoid using absolute positionning with random values to avoid breaking on diffrent screen sizes
- Here is a simple example to position the image:

```tsx
<div className="absolute inset-0 z-10 mt-4">
  <Image src="/images/womens.webp"
    fill
    alt="womens"
    className="object-contain"
  />
</div>

```

Rofieda-Maamar committed:
fix: update awareness section images and text

belaid1meriem committed:
Refactor AwarenessSection layout and update image sources

- Removed unnecessary image properties and simplified the layout of the AwarenessSection component.
- Updated image sources in awareness-data to use SVG files for improved quality and responsiveness.

belaid1meriem committed:
Remove unused awareness images and refactor awareness data structure to simplify image handling

belaid1meriem committed:
Merge branch 'main' into feature/awareness
